application/event_processor/event_processor.py

```diff
@@ -18,7 +18,12 @@
 from common.middlewares import set_correlation_id, unhandled_exception_logging
 from common.utilities import extract_body, get_sequence_number
 from nhs import NHSEntity
-from reporting import log_invalid_open_times, log_unmatched_nhsuk_pharmacies, report_closed_or_hidden_services
+from reporting import (
+    log_invalid_open_times,
+    log_un_matched_service_types,
```

Comment:
I think `log_un_matched_service_types` should be named `log_unmatched_service_types`, I think unmatched is one word.